docs/devel.md

```diff
@@ -33,3 +33,16 @@ now the informers:
 ```shell
  ${GOPATH}/bin/informer-gen --input-dirs "github.com/amadeusitgroup/workflow-controller/pkg/api/workflow/v1" --versioned-clientset-package "github.com/amadeusitgroup/workflow-controller/pkg/client/versioned" --listers-package "github.com/amadeusitgroup/workflow-controller/pkg/client/listers" --output-package "github.com/amadeusitgroup/workflow-controller/pkg/client/informers"   -h "hack/boilerplate.go.txt"
 ```
+
+
+# How to generate api for cron workflows
+
+```shell
+$ go get k8s.io/code-generator/...
```

Comment:
the problem with this method is that you don't used a specific version the code-generator. And so someone who will try to regenerate the files, may not have the same result.
Personally, I put the `code-generator` as glide dependencies.

Reply:
I know and it's in my TODO list

Reply:
```yaml
- package: k8s.io/code-generator
  version: kubernetes-1.9.1
```